.gitignore

```diff
@@ -10,6 +10,7 @@
 /src/Sulu/Bundle/*/Tests/app/cache
 /src/Sulu/Bundle/*/Tests/app/logs
 /src/Sulu/Bundle/*/Tests/app/data
+/src/Sulu/Bundle/TestBundle/Resources/app/cache
```

Comment:
As already said I would prefer to have this in the `.gitignore` file of the TestBundle itself, since a potential subtree split in the future might make it harder to maintain these files.